### DISCUSSION TITLE
How to open the assets manager on clicking image in html

### DISCUSSION THREAD

**Discussion by Pulkitp44** [GrapesJS/grapesjs#4200] [Q&A]
Hi@artf
#1155 
I have made a component as explained now I want to open the assets manager when someone click on image defined in component to change the default image which was provided in block created 

Thanks

**Comment by artf** [comment]
Check the [image view implementation](https://github.com/artf/grapesjs/blob/dev/src/dom_components/view/ComponentImageView.js). There is the `onActive` method which is triggered on `events.dblclick` (the onActive is also triggered when you define a block with the [activate option](https://grapesjs.com/docs/modules/Blocks.html#initialization))
